examples/go-code-tutorial/README.md

```diff
@@ -0,0 +1,12 @@
+# Go Rules Tutorial
+
+This is the source code for the Go Rules tutorial that is available here:
+https://bazel-contrib.github.io/SIG-rules-authors/go-tutorial.html
+
+## Updates
```

Comment:
How about moving this part to a separate file called `CONTRIBUTING.md` or `DEVELOPERS.md`? It's meant for rules_go developers, not for the typical audience of this tutorial.